tests/e2e/README.md

```diff
@@ -86,3 +86,18 @@ For running tests without docker, please perform next steps:**
 **Run command:**
 
 - npm run test-plugin
+
+
+## Launching the DevWorkspaceHappyPath spec file using Che with oauth authentication
+
+**Setup next environment variables:**
+
+- export TS_SELENIUM_BASE_URL=<Che-URL\>
+- export TS_SELENIUM_OCP_USERNAME=<cluster-username\>
+- export TS_SELENIUM_OCP_PASSWORD=<cluster-password\>
+- export TS_SELENIUM_VALUE_OPENSHIFT_OAUTH="true"
+- export TS_OCP_LOGIN_PAGE_PROVIDER_TITLE=<login-page-provider-link>
```

Comment:
it's not about link
It's name of identity provider expressed at the login page

Reply:
Also, please make sure info is displayed correctly:
![Screenshot from 2022-03-10 15-25-14](https://user-images.githubusercontent.com/1197777/157671032-5edc3274-1040-45be-bb16-704874f306c6.png)

Reply:
Updated